Ecosystem Partners/getting-started-with-vormetric-dsm.md

```diff
@@ -53,10 +53,15 @@ This deployment process for Partner Templates currently requires manual interact
 ### Postrequisite
 - If you want to access your Vormetric DSM over the internet, please perform the following tasks once your Vormetric DSM has been deployed to your account:
 1. Adding an external ip-address to an existing or new CenturyLink server
-![ip.jpg](https://t3n.zendesk.com/attachments/token/kObGC9P2IjP1ate0NexwFNiXz/?name=ip.jpg)
```

Comment:
The text in between the brackets is for img alt. The purpose of alt text is for screen readers to describe what the image is of. To a blind person, having a screen reader say "ip.jpg" isn't all that useful ;). Can you fix these so that it doesn't just display the image file name?